test/Dockerfile

```diff
@@ -1,6 +1,4 @@
-# Uses focal image because newer jammy image does not work with old docker versions
-# see related information: https://github.com/adoptium/containers/issues/215#issuecomment-1142046045
-FROM mambaorg/micromamba:1.4.2-focal
+FROM mambaorg/micromamba:1.5.8-jammy
```

Comment:
This updates the base image, but the old version was only because we couldn't update docker on the bamboo agents.

Reply:
It does seem to fix some vulnerabilities as well. 
<img width="985" alt="Screenshot 2024-08-05 at 1 46 31 PM" src="https://github.com/user-attachments/assets/7282cf35-e1ee-433e-a562-999e36b0a913">
<img width="1019" alt="Screenshot 2024-08-05 at 1 45 46 PM" src="https://github.com/user-attachments/assets/6aca5ea8-30b0-436c-af84-525c319e7d16">